### PR TITLE
Create scripts for building and pushing psql image with schema loaded

### PR DESCRIPTION
**Pull request by kimlisa**
New files are located in `./dockerfiles/hwsc-user-svc`

`build_push_psql_image.sh`: Builds and creates a postgres image with a database created and schema inserted. Combines all *.up.sql scripts into one sql file.

### CODE REVIEW

File: dockerfiles/hwsc-user-svc/build_push_psql_image.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# combine all *.up.sql files into one sql file for docker build to consume
+cat ../../db-migrations/hwsc-user-svc/test/psql/*.up.sql > user_svc_schema.sql
+
+# build an image from a docker file, -t = name and optionally a tag in the 'name:tag' format
+# docker build [OPTIONS] PATH
+docker build --build-arg db_name=$hosts_postgres_db -t hwsc/hwsc-user-svc-psql:int .
```
Comment (faraonc) [inline dockerfiles/hwsc-user-svc/build_push_psql_image.sh:8]: This should be tagged `:test-int`.

Comment (faraonc) [reply dockerfiles/hwsc-user-svc/build_push_psql_image.sh:8]: `:int` tag will be used for deployed dev environment

**Review by faraonc — COMMENTED** [review]
Just some tagging clarification.

File: dockerfiles/hwsc-user-svc/build_push_psql_image.sh (hunk 1 above)
Comment (kimlisa) [reply dockerfiles/hwsc-user-svc/build_push_psql_image.sh:8]: Pushed changes to master and deleted :int tag from docker hub

**Pull request merged**